packages/sync-service/lib/electric/plug/label_process_plug.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Electric.Plug.LabelProcessPlug do
+  @moduledoc """
+  A plug that assists debugging by labelling processes that handle requests with
+  details about the request.
+
+  The plug should be places right after the match plug in the router:
+
+    plug :match
+    plug Electric.Plug.LabelProcessPlug
+  """
+
+  def init(opts), do: opts
+
+  def call(conn, _opts) do
+    conn
+    |> process_label()
```

Comment:
Maybe pull request ID from the conn too and use that in the label as well? Generally all logs ought to be correlatable by the request ID, not by full path, right?

Reply:
Ooh, good idea. I'll add it.

Reply:
Done